packages/kbn-shared-ux-components/src/page_template/page_template.mdx

```diff
@@ -0,0 +1,9 @@
+--
+id: sharedUX/Components/PageTemplate
+slug: /shared-ux-components/page_template/page_template
+title: Page Template
+summary: A Kibana-specific wrapper around `EuiTemplate`
+tags: ['shared-ux', 'component']
+date: 2022-04-04
+---
+This component is a thin wrapper around `EuiTemplate`, providing Kibana-specific additions. It can be configured to display solution navigation bar, empty page and no data config page. 
```

Comment:
Since we're mostly copying right now, can we copy/paste the existing doc for KibanaPageTemplate into this file and update any references/screenshots?

https://github.com/elastic/kibana/blob/c4815d319ecc4894315d7b82a3af9066aa3e5dbb/dev_docs/tutorials/kibana_page_template.mdx

Reply:
I have updated the .mdx file and made corresponding changes where needed (omitting "recommended" and changing the `actions` to `action`). We might want to update that last screenshot though, as it still shows 2 cards.

Reply:
🙏  Thank you! Here's an updated image:

![image](https://user-images.githubusercontent.com/549577/162009478-a13ff25e-b1be-42ea-83ef-55ecc9355571.png)

Reply:
Images are loaded from `dev_docs`, so I opened a separate [PR](https://github.com/elastic/kibana/pull/129641) for that (the URL remains the same).